### ISSUE TITLE
Avoid committing static builds

### ISSUE THREAD

**Issue by derekrushforth** [wildbit/beanstalk-code-snippet-bot#5]
The node container on Beep Boop doesn't work with ES6 runtime compiling using `babel-register`. To work around this I re-enabled static JS builds but it's super messy committing directly to the repo.

It would be nice to figure out a way that we can avoid this and build directly on the container.

**Comment by derekrushforth** [comment]
You may already be aware of this. But it looks like a8025c6e9a914444a68f1c8855e05bdc788eb83e is causing this build error on the container.

```
[   2016-05-02T16:56:19Z    ]   npm info beanstalk-code-snippet-bot@1.0.0 Failed to exec prestart script
[   2016-05-02T16:56:19Z    ]   npm ERR! Linux 3.16.0-4-amd64
[   2016-05-02T16:56:19Z    ]   npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
[   2016-05-02T16:56:19Z    ]   npm ERR! node v4.3.1
[   2016-05-02T16:56:19Z    ]   npm ERR! npm v2.14.12
[   2016-05-02T16:56:19Z    ]   npm ERR! code ELIFECYCLE
[   2016-05-02T16:56:19Z    ]   npm ERR! beanstalk-code-snippet-bot@1.0.0 prestart: `npm run build`
[   2016-05-02T16:56:19Z    ]   npm ERR! Exit status 1
[   2016-05-02T16:56:19Z    ]   npm ERR!
[   2016-05-02T16:56:19Z    ]   npm ERR! Failed at the beanstalk-code-snippet-bot@1.0.0 prestart script 'npm run build'.
[   2016-05-02T16:56:19Z    ]   npm ERR! This is most likely a problem with the beanstalk-code-snippet-bot package,
[   2016-05-02T16:56:19Z    ]   npm ERR! not with npm itself.
[   2016-05-02T16:56:19Z    ]   npm ERR! Tell the author that this fails on your system:
[   2016-05-02T16:56:19Z    ]   npm ERR! npm run build
[   2016-05-02T16:56:19Z    ]   npm ERR! You can get their info via:
[   2016-05-02T16:56:19Z    ]   npm ERR! npm owner ls beanstalk-code-snippet-bot
[   2016-05-02T16:56:19Z    ]   npm ERR! There is likely additional logging output above.
[   2016-05-02T16:56:19Z    ]   
[   2016-05-02T16:56:19Z    ]   npm ERR! Please include the following file with any support request:
[   2016-05-02T16:56:19Z    ]   npm ERR! /usr/src/app/npm-debug.log
```

**Comment by okonet** [comment]
Yeah. I was trying to fix it without compiling locally. Still on it.

**Comment by okonet** [comment]
I've migrated to Node6 in order to resolve this.

**Comment by okonet** [comment]
Closed with #6

**Comment by derekrushforth** [comment]
Looks good!
